eks/modules/services/k8s.tf

```diff
@@ -15,4 +15,27 @@ provider "kubernetes" {
 }
 
 //To do
-// Add a storage class resource with name "persistent"
\ No newline at end of file
+// Add a storage class resource with name "persistent"
+resource "kubernetes_storage_class" "persistent" {
+  metadata {
+    name = "persistent"
+  }
+  storage_provisioner = "kubernetes.io/aws-ebs"
+  reclaim_policy      = "Delete"
+  allow_volume_expansion = true
+  parameters = {
+    type = "gp2"
+  }
+}
+resource "kubernetes_storage_class" "scopedpersistent" {
+  metadata {
+    name = "scopedpersistent"
+  }
+  storage_provisioner = "kubernetes.io/aws-ebs"
+  reclaim_policy      = "Retain"
+  allow_volume_expansion = true
+  parameters = {
+    type = "gp2"
+    zone = "${var.storage_zones}"
+  }
+}
```

Comment:
Thanks! A non-default storage class config was missing...

Reply:
You are always welcome :)